glib/src/subclass/object.rs

```diff
@@ -848,7 +661,7 @@ mod test {
         .expect("Failed to connect on 'create-child-object'");
 
         let value = obj
-            .emit("create-child-object", &[])
+            .emit_by_name("create-child-object", &[])
```

Comment:
We could make use of `emit()` here in one place or another to increase test coverage a bit (but also keep calling `emit_by_name()` somewhere for the same reason)